addons/mass_mailing/controllers/main.py

```diff
@@ -180,15 +181,11 @@ def _mailing_unsubscribe_from_list(self, mailing, document_id, email, hash_token
         mailing.contact_list_ids._update_subscription_from_email(email, opt_out=True)
         # compute name of unsubscribed list: hide non public lists
         if all(not mlist.is_public for mlist in mailing.contact_list_ids):
-            lists_unsubscribed_name = _('You are no longer part of our mailing list(s).')
+            lists_unsubscribed_name = False
         elif len(mailing.contact_list_ids) == 1:
-            lists_unsubscribed_name = _('You are no longer part of the %(mailing_name)s mailing list.',
-                                        mailing_name=mailing.contact_list_ids.name)
+            lists_unsubscribed_name = mailing.contact_list_ids.name
         else:
-            lists_unsubscribed_name = _(
-                'You are no longer part of the %(mailing_names)s mailing list.',
-                mailing_names=', '.join(mlist.name for mlist in mailing.contact_list_ids if mlist.is_public)
-            )
+            lists_unsubscribed_name = ', '.join(mlist.name for mlist in mailing.contact_list_ids if mlist.is_public)
```

Comment:
so if I have one list, it can not be public, but if I have more, they we only take the public ones ?

Shouldn't the `elif` case be removed altogether ? What is the reasoning here ?